tests/test_server.py

```diff
@@ -1749,7 +1750,8 @@ def test_get_output_channel(
 ):
     request = MagicMock()
     app = MagicMock()
-    app.input_channels = input_channels
+    app.ctx = Namespace()
```

Comment:
> Looks great! Would be great to confirm this works on the repo that flagged this issue with a manual test too 🚀

Yes. I did this

Reply:
Thank you for the explanation, that makes sense 🙏🏼  
The only suggestion I have is to condense this in one line, e.g. `app = MagicMock(ctx=Namespace())` since `MagicMock` accepts arbitrary keyword args.